tools/iothub-explorer/iothub-explorer.js

```diff
@@ -395,6 +411,9 @@ function usage() {
     '    {grey}Sends a cloud-to-device message to the given device, optionally with acknowledgment of receipt{/grey}',
     '  {green}iothub-explorer{/green} {white}[<connection-string>] receive [--messages=n]{/white}',
     '    {grey}Receives feedback about the delivery of cloud-to-device messages; optionally exits after receiving {white}n{/white} messages.{/grey}',
+    '  {green}iothub-explorer{/green} {white}[<connection-string>] sas-token <device-id> [expiry]{/white}',
```

Comment:
I'd recommend that this command have a --duration option that mimics the one for the 'login' command (in seconds, one hour default if option isn't specified)